README.rst

```diff
@@ -14,6 +14,12 @@ A comprehensive list of changes in each version may be found in the `CHANGELOG`_
 - `Client Library Documentation`_
 - `github.com/googleapis/python-storage`_
 
+Certain control plane and long-running operations for Cloud Storage (including Folder
+and Managed Folder operations) are supported via the `Storage Control Client`_.
+The `Storage Control API`_ creates one space to perform metadata-specific, control plane,
+and long-running operations apart from the Storage API. Separating these operations from
+the Storage API improves API standardization and lets you run faster releases.
```

Comment:
> Separating these operations from the Storage API improves API standardization and lets you run faster releases.

Should this sentence be omitted? If not, can you clarify the `lets you run faster releases` part?

Reply:
Updated, thanks!